images/universal/training/README.md

```
@@ -0,0 +1,219 @@
+# Universal Training Images
+
+Universal images that serve dual purposes: **Jupyter Workbench** (default) and **Training Runtime** (when command provided). These images are designed for FIPS compliance (best effort) and hermetic builds in downstream Konflux pipelines.
+
+These images are structured to support **hermetic builds** in downstream Konflux/Tekton pipelines:
+
+### Midstream vs Downstream
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)